packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -508,7 +524,7 @@ sdk.server('https://eu.api.example.com/v14');`)
     paramTypes?: OperationTypeHousing['types']['params'],
     responseTypes?: OperationTypeHousing['types']['responses']
   ) {
-    const docblock: OptionalKind<JSDocStructure> = { tags: [] };
+    const docblock: OptionalKind<JSDocStructure> = {};
```

Comment:
I made all these changes to docblock generation because I discovered while building this test spec for problematic schema titles that if an operation has no `summary` or `description` we would give it an empty `/** */` docblock.